Marlin/UltiLCD2.cpp

```diff
@@ -290,14 +291,14 @@ static void lcd_menu_breakout()
     {
         uint8_t x = (ball_x >> 8) / 25;
         uint8_t y = (ball_y >> 8) / 10;
-        if (lcd_cache[x+y*5])
+        if (breakout_buf[x+y*5])
```

Comment:
Might be an (optimization) idea to pull the semi-constant x+y*5 out into a separate index variable since it is being used nearly everywhere?

Reply:
It is less constant than it seems at first sight. Also, this code is an easter egg for which I don't want to waste time in optimizations which would require extra testing too.